nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestCommentManager.java

```diff
@@ -99,10 +97,15 @@
 
     @Before
     public void init() {
-        DocumentModel domain = session.createDocumentModel("/", "Folder", "Folder");
+        DocumentModel domain = session.createDocumentModel("/", "domain", "Domain");
+        session.createDocument(domain);
+        doc = session.createDocumentModel("/domain", "test", "File");
+        doc = session.createDocument(doc);
+
+        domain = session.createDocumentModel("/", "Folder", "Folder");
```

Comment:
Name it `folder`, or reuse `doc` maybe? `domain` is misleading.

Reply:
I was indeed trying to picky spare resource. I made a new `DocumentModel folder` to be clearer